src/Elements/Serialization/glTF/GltfExtensions.cs

```diff
@@ -676,11 +676,14 @@ private static Gltf InitializeGlTF(Model model, List<byte> buffer, bool drawEdge
 
             var materialsToAdd = model.AllElementsOfType<Material>().ToList();
             materialsToAdd.Add(BuiltInMaterials.Edges);
+            materialsToAdd.Add(BuiltInMaterials.Default);
             var materials = gltf.AddMaterials(materialsToAdd, buffer, bufferViews);
 
             var elements = model.Elements.Where(e =>
             {
-                return e.Value is GeometricElement || e.Value is ElementInstance;
+                return e.Value is GeometricElement || 
+                e.Value is ElementInstance ||
+                e.Value is ITessellate;
```

Comment:
Will there be any ITessellates that aren't also GeometricElements?

Reply:
This is a good catch. This was added before we made `AnalysisMesh` a `GeometricElement`. So the short answer to this question is no.